onnxruntime/contrib_ops/cpu/transformers/logits_processor.h

```diff
@@ -76,12 +76,24 @@ class VocabMaskLogitsProcessor : public ILogitsProcessor<T> {
   gsl::span<const int32_t> vocab_mask_;
 };
 
+template <typename T>
+class PrefixVocabMaskLogitsProcessor : public ILogitsProcessor<T> {
```

Comment:
No need add a new class. I think you only need modify VocabMaskLogitsProcessor (like add a Boolean parameter in constructor etc.) since most logic is same.

Reply:
Updated the current class.